src/skins/vector/css/matrix-react-sdk/views/messages/_MFileBody.scss

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright 2015, 2016 OpenMarket Ltd
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+.mx_MFileBody_download {
+    color: $accent-color;
+}
+
+.mx_MFileBody_download a {
+    color: $accent-color;
+    text-decoration: none;
+    cursor: pointer;
```

Comment:
I should have done the file splitting and the adding of the new css in two commits - sorry
AAAAAAnyway, the above line is the only one added